source/includes/_examples.md

```diff
@@ -0,0 +1,219 @@
+# Examples
+
+## CDC Guideline for Prescribing Opioids for Chronic Pain - Request
```

Comment:
I just tested both `##` (h2) and `###` (h3) and they work. Only `#` (h1) and `##` headers will show in the left hand sidebar. h3 headers do show in the middle documentation panel.